integration-tests/observability-lgtm/src/main/resources/application.properties

```diff
@@ -0,0 +1,14 @@
+# Disable default binders
+quarkus.micrometer.binder-enabled-default=false
+
+quarkus.log.category."io.quarkus.observability".level=DEBUG
+
+#micrometer
+quarkus.micrometer.export.otlp.enabled=true
+quarkus.micrometer.export.otlp.publish=true
+quarkus.micrometer.export.otlp.step=PT5S
+quarkus.micrometer.export.otlp.default-registry=true
+quarkus.micrometer.export.otlp.url=http://${quarkus.otel-collector.url}/v1/metrics
```

Comment:
Should we standardise around HTTP (4318) port?

Reply:
OK, set 4318 as default